src/V3Ast.cpp

```diff
@@ -33,8 +33,12 @@ VL_DEFINE_DEBUG_FUNCTIONS;
 //======================================================================
 // Statics
 
+#ifdef VL_DEBUG
 uint64_t AstNode::s_editCntLast = 0;
 uint64_t AstNode::s_editCntGbl = 0;  // Hot cache line
+#else
+bool AstNode::s_editedGbl = false;
```

Comment:
```suggestion
bool AstNode::s_editedGbl = false;  // Hot cache line
```

I probably would have kept s_editCntGbl as a uint64_t so less conditional code, I suspect performance is similar, but I don't object to this way.